src/floating_label.cpp

```diff
@@ -104,6 +104,11 @@ rect floating_label::get_bg_rect(const rect& text_rect) const
 	};
 }
 
+void floating_label::clear_texture()
+{
+	tex_ = {};
```

Comment:
There is also a `reset` function.

https://github.com/wesnoth/wesnoth/blob/25918997231496656a9c00ae45c61638346fcecd/src/sdl/texture.hpp#L187-L188